skrub/_gap_encoder.py

```diff
@@ -566,8 +629,13 @@ class GapEncoder(TransformerMixin, BaseEstimator):
     rescale_W : bool, default=True
         If `True`, the weight matrix `W` is rescaled at each iteration
         to have a l1 norm equal to 1 for each row.
-    max_iter_e_step : int, default=20
+    max_iter_e_step : int, default=1
         Maximum number of iterations to adjust the activations h at each step.
+    max_no_improvement : int, default=5
+        Control early stopping based on the consecutive number of mini batches
+        that does not yield an improvement on the smoothed cost function.
+        To disable convergence detection based on cost function,
+        set max_no_improvement to None.
```

Comment:
It's not be obvious what happens then when `max_no_improvement=None`. Maybe this is clearer:
```suggestion
        To disable early stopping and run the process fully,
        set ``max_no_improvement=None``.
```